### ISSUE TITLE
Support Android 2.1 or later

### ISSUE THREAD

**Issue by JRHaigh** [gabm/TapAndTurn#12]
Tap 'n' Turn is brilliant! As such, I'd like to use it on all of my Android devices. Just because I have newer devices doesn't make the old ones any less functional, so please avoid encouraging obsolescence. Seeing as [Rotation Manager](https://f-droid.org/app/com.spydiko.rotationmanager_foss) supports Android 2.1 or later, it seems reasonable to request that you try to match this.
&nbsp;&nbsp;&nbsp;&nbsp;Ever since my first smartphone I've realised that Android needed something better than a stupid autorotate switch, especially seeing as rotations of 90° trigger a lot of bugs in a lot of applications, so the common accidental rotations associated with autorotation actually risk loss of data and application crashes. However, I never got round to seeing whether I could do anything about implementing the ideas I had (see #11), especially seeing as Rotation Manager became available, providing an adequate workaround for the applications that I realised were prone to losing data or crashing upon rotation.
&nbsp;&nbsp;&nbsp;&nbsp;Being GPLv3, like Rotation Manager, Tap 'n' Turn is likely to make it onto [my list of recommendations for Android](https://wiki.fsfe.org/Fellows/JRHaigh/RFA) when it has matured a bit. If like Rotation Manager it can be compatible with 2.1 or later, thus supporting all of my Android devices, and runs at boot, this will likely replace Rotation Manager in my recommendations. Thanks again for your work!

**Comment by gabm** [comment]
yes, sounds reasonable. Didn't give it much importance by now, but as it's so simple, we'll probably not have big issues..

Can you have a look into this? e.g. just lower the minSdk to android 2.1 (e.g. api level 7 or something like that) and check what still works/builds and what doesn't?!

**Comment by gabm** [comment]
I started looking into this and started creating a [Compatibility Chart](https://github.com/gabm/TapAndTurn/wiki/Compatibility-Chart).  Please feel free to extend it...

Currently the `Switch` ui element is keeping us from lowering the api version below 14. Do you want to take a look into replacing it with something more compatible?

**Comment by Madis0** [comment]
My two cents: while this app is quite simple by idea, I don't think it is worth it to maintain a backport, especially when the developer lacks time and there are other issues to solve. 

It would be much better to target Oreo (26-27) instead to support new and upcoming devices. OP could try [Adaptive Rotation Lock](https://play.google.com/store/apps/details?id=ui.robot.rotate), it supports even Froyo (API 8).
